### PR TITLE
with_output should be False in run_sorters

### PR DESCRIPTION
**Pull request by yger**
When launching run_sorters in a study, one should not collect outputs, such that if there is a crash of any sorters, the loop keeps working. Plus, this is redondant with copy_sorters()

### CODE REVIEW

File: spikeinterface/sorters/runsorter.py [hunk 1]
```diff
@@ -45,7 +45,7 @@
 def run_sorter(sorter_name, recording, output_folder=None,
                remove_existing_folder=True, delete_output_folder=False,
                verbose=False, raise_error=True, docker_image=None,
-               **sorter_params):
+               with_output=True, **sorter_params):
```
Comment (samuelgarcia) [inline spikeinterface/sorters/runsorter.py:48]: Maybe you should add some test like this
```python
if not with_output and remove_existing_folder:
    raise Exception('If you delete the output folder and do not retrrieve result, you are a bad boy')

```

**Comment by samuelgarcia** [comment]
Hi Pierre. can you resolve conflicts ?

**Comment by samuelgarcia** [comment]
Oups sorry I close it accidentaly.

**Comment by yger** [comment]
No problem. The PR is back, and solved all issues encountered with run_sorters and YASS:
* fix for the docker
* do not crash if a sorter crash
* works with docker images

**Comment by samuelgarcia** [comment]
OK for me.
@alejoe91 : can you also validate this ?

**Comment by samuelgarcia** [comment]
OK lets merge.

**Pull request merged**